client.go

```diff
@@ -36,8 +36,11 @@ const (
 // Client is the API client for Targetprocess. Create this using NewClient.
 // This can also be constructed manually but it isn't recommended.
 type Client struct {
-	// BaseURL is the base URL for API requests.
-	BaseURL *url.URL
+	// baseURL is the base URL for v1 API requests.
+	baseURL *url.URL
```

Comment:
I see why this is unexported, but can you envision any scenario in which the consumer might want to access this? Possibly to make custom http requests to the URL that aren't covered by this library?

Reply:
I guess that's covered by the generic `GET` method, so probably not an issue.